display_list/display_list.cc

```diff
@@ -180,6 +182,14 @@ static bool CompareOps(uint8_t* ptrA,
   return true;
 }
 
+void DisplayList::RenderTo(DisplayListBuilder* builder,
+                           SkScalar opacity) const {
+  if (!builder) {
+    return;
+  }
+  Dispatch(*builder);
```

Comment:
TODO: deal with attributes and opacity. Perhaps file an issue and I can deal with it.

I don't think the quick and dirty code I provided in the chat would work here, something slightly more involved will be required (like a saveAttributes/restore type of thing).